spec/models/landing_page/block/side_navigation_spec.rb

```diff
@@ -1,24 +1,17 @@
 RSpec.describe LandingPage::Block::SideNavigation do
   let(:blocks_hash) do
-    { "type" => "side_navigation" }
-  end
-  let(:subject) { described_class.new(blocks_hash, build(:landing_page)) }
-
-  before do
-    LandingPage::Block::SideNavigation.send(:remove_const, "LINKS_FILE_PATH")
-    LandingPage::Block::SideNavigation.const_set("LINKS_FILE_PATH", "spec/fixtures/landing_page/links/side_navigation.yaml")
-  end
-
-  after do
-    LandingPage::Block::SideNavigation.send(:remove_const, "LINKS_FILE_PATH")
-    LandingPage::Block::SideNavigation.const_set("LINKS_FILE_PATH", "lib/data/landing_page_content_items/links/side_navigation.yaml")
+    {
+      "type" => "side_navigation",
+      "navigation_group_id" => "Submenu",
+    }
   end
+  let(:subject) { described_class.new(blocks_hash, build(:landing_page_with_navigation_groups)) }
 
   describe "#links" do
-    it "returns all of the side navigation links" do
+    it "returns all of the navigation links from the requested menu" do
```

Comment:
I think "from the requested menu" should be removed from the test name. or changed to "from the requested group".

Reply:
Done.